### ISSUE TITLE
Intuit

### ISSUE THREAD

**Issue by diegobot1977** [simple-icons/simple-icons#5894]
**Brand Name:** Intuit
**Website:** https://www.intuit.com/
**Alexa rank:** 97
**Official resources for icon and color:** Also used from _this color_ of the logo - `#0076c9` but used the third letter of `T`. See Below in the Intuit homepage

**Comment by diegobot1977** [comment]
Hey @service-paradis! I got this one for this Intuit icon:

![intuit](https://user-images.githubusercontent.com/73037261/121396195-1fd3a580-c921-11eb-908e-b72f62972855.png)

I go this third letter `T` for this Intuit icon, and just wrote in SVG-edit.

**Comment by diegobot1977** [comment]
Hey @service-paradis! Consider previewing this one:

![intuit](https://user-images.githubusercontent.com/73037261/121396195-1fd3a580-c921-11eb-908e-b72f62972855.png)

I go this of third letter `T` for this Intuit icon, and just wrote in SVG-edit.

**Comment by diegobot1977** [comment]
Also consider previewing this one:

![intuit](https://user-images.githubusercontent.com/73037261/121396195-1fd3a580-c921-11eb-908e-b72f62972855.png)

I go this of third letter `T` for this Intuit icon, and just wrote in SVG-edit.

**Comment by diegobot1977** [comment]
Oh sure!

**Comment by diegobot1977** [comment]
Well I don't know what should I do my pull request :|

**Comment by diegobot1977** [comment]
Sorry, I just don't know where should I create a pull request to add this icon...

**Comment by fbernhart** [comment]
@diegobot1977 Just like you already did. You didn't do anything wrong.

Just DON'T close it! 😄

I'll reopen it for you.

**Comment by diegobot1977** [comment]
Thanks.

**Comment by diegobot1977** [comment]
We'll, let's wait for others to create a pull request for adding an Intuit icon.

**Comment by ericcornelissen** [comment]
Closing this in favor #5861. @diegobot1977, please open issues for icon requests only once in the future. If you need help with GitHub and creating  a  Pull  Request do feel free to ask for help in one of your issues or [as a  discussion topic](https://github.com/simple-icons/simple-icons/discussions).
